packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/index.md

```diff
@@ -113,66 +114,119 @@ For password-only authentication, you need to update the password authenticator
-1. Select the **Enrollment** tab, and then click **Add Multifactor Policy**.
-1. Give the new policy a name. For example, "Email sign-up required policy".
-1. Set **Assign to groups** to one or more groups. For example, "Everyone".
-1. In the **Eligible Authenticators** section
-   * Set **Email** to **Required**.
-   * Set **Password** to **Optional**.
-   * Verify that the remaining authenticators are set to **Optional**.
-1. Click **Create Policy**.
-1. Give the rule a name. For example, "Email sign-up required for all".
-1. Leave the other settings at their defaults, and then click **Create Rule**.
-
-#### Add a new Global session policy
+2. Select the **Enrollment** tab, and then click **Add Multifactor Policy**.
+3. Give the new policy a name. For example, "Password-optional Sign-In Policy".
+4. Set **Assign to groups** to the group you just made for password-optional users.
+5. In the **Eligible Authenticators** section:
 
-1. Open the **Admin Console** for your org.
-1. Choose **Security** > **Global Session Policy**.
-1. Click **Add policy**.
-1. Give the policy a name. For example, "Global Password Optional Policy".
-1. Set **Assign to groups** to one or more user groups. For example, "Everyone".
-1. Click **Create Policy and Add Rule**.
-1. Give the rule a name. For example, "Global Password Optional Rule".
-1. Verify **Establish the user session with** is set to **Any factor used to meet the Authentication Policy requirements**.
-1. Leave the other settings at their defaults, and then click **Create Rule**.
-1. Click **Create Rule**.
-1. Verify the new policy is in the number "1" position. If it isn't, click and drag the policy to that position.
+   1. Set **Email** to **Required**.
+   2. Set **Password** to **Optional**.
+   3. Verify that the remaining authenticators are set to **Optional**.
 
-#### Add a new authentication policy
+6. Click **Create Policy**.
+7. Give the rule a name. For example, "Password-optional Sign-In Rule".
+8. Set **Exclude Users** to the names of your main admin accounts.
+9. Leave the other settings at their defaults, and then click **Create Rule**.
+10. Move the new policy immediately above the Default Policy in the list of policies.
+
+##### Add a new global session policy for password-optional users
+
+A global session policy determines user session length and basic authentication rules for groups of users. In this case, the policy turns off MFA for all users in the password-optional user group. Therefore, they only need email authentication to sign in.
+
+1. Choose **Security** > **Global Session Policy**.
+2. Click **Add policy**.
+3. Give the policy a name. For example, "Global Password Optional Policy".
+4. Set **Assign to groups** to the group you just made for password-optional users.
+5. Click **Create Policy and Add Rule**.
+6. Give the rule a name. For example, "Global Password Optional Rule".
+7. Set **Exclude Users** to the names of your main admin accounts
```

Comment:
We probably don't need this step since the admin accounts won't be in the new  "password-optional users" group.

Reply:
fixed